llap-server/src/test/org/apache/hadoop/hive/llap/cache/TestBuddyAllocator.java

```diff
@@ -87,7 +91,7 @@ public void testSameSizes() throws Exception {
     int min = 3, max = 8, maxAlloc = 1 << max;
     BuddyAllocator a = new BuddyAllocator(isDirect, isMapped, 1 << min, maxAlloc, maxAlloc,
         maxAlloc, 0, tmpDir, new DummyMemoryManager(),
-        LlapDaemonCacheMetrics.create("test", "1"), null, true);
+        LlapDaemonCacheMetrics.create("test", "1"), null, true, 1024 * 1024 * 128);
```

Comment:
You're using this 128MB value multiple times in this suite. Maybe, you want to move this into a const.